### DISCUSSION TITLE
What happens if user changes their email in 3rd party oauth provider?

### DISCUSSION THREAD

**Discussion by tombburnell** [nextauthjs/next-auth#4172] [Help]
If a user logs in with facebook and you create an account based on their email address, what happens if they change their facebook email address? Would this not cause their account to become orphaned and they would get a empty account next time they logged in?

How do people typically handle this?

With google, it's a fixed email so it's fine but with services that you can change your email this must be a problem right?

Thanks

**Comment by choutkamartin** [comment]
Nothing should happen. To my knowledge, operations towards the Account model are made based on the `providerAccountId`, which is an ID each provider should return. This ID shouldn't change.
